src/styles/components/_details-page.scss

```diff
@@ -0,0 +1,42 @@
+.details-page {
+  overflow: hidden;
+
+  &__header {
+    width: 100%;
+    margin: auto;
+    background-image: url("../../../public/background_image.png");
+  }
+
+  &__banner {
+    width: 100%;
+    margin: auto;
+    background-color: map.get($colors, "accent-teal");
+  }
+
+  &__primary {
+    width: 100%;
+    margin: auto;
+    padding: 1.25rem;
+    max-width: 100rem; // 1600px;
+    overflow: hidden;
+  }
+
+  &__secondary {
+    position: relative;
+    width: 100%;
+    margin: auto;
+    padding: 3.75rem 1.5rem;
```

Comment:
Looks like in the Figma for mobile, we want this padding to be 20px/1.25rem, primary and secondary sections line up for mobile
<img width="355" alt="image" src="https://user-images.githubusercontent.com/54716846/200096803-caf89ee6-c2ee-4d2d-9b33-90d564d3a5e3.png">

<img width="393" alt="image" src="https://user-images.githubusercontent.com/54716846/200096836-b8da1093-abeb-41ad-827d-733c43b14c93.png">